tests/TestTCP.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE RebindableSyntax, TemplateHaskell #-}
+{-# LANGUAGE BangPatterns #-}
```

Comment:
No just hard to spot: line 786. Just wanted to be sure strictness is OK. Maybe this doesn't need to be a part of the test suite.